include/mbedtls/ssl.h

```diff
@@ -1184,6 +1184,11 @@ struct mbedtls_ssl_session {
     unsigned char MBEDTLS_PRIVATE(mfl_code);     /*!< MaxFragmentLength negotiated by peer */
 #endif /* MBEDTLS_SSL_MAX_FRAGMENT_LENGTH */
 
+/*!< Record Size Limit for outgoing data requested by peer */
```

Comment:
```suggestion
/*!< RecordSizeLimit received by peer */
```

Reply:
This makes it sounds as if the added field is the limit the peer is going to use for data sent to us.

Reply:
Done.